src/main/steps/applicant/task-list/content.test.ts

```diff
@@ -90,9 +90,9 @@ describe('task-list > content', () => {
         {
           items: [
             {
-              href: '/applicant/keep-details-private/details_known',
+              href: '/applicant/yourhearings/hearings',
```

Comment:
are you sure we need this change?

Reply:
When the user logs in as applicant, he will require this href to view the hearings